scripts/ufm_ports/load_ports.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python3
+
+"""
+@copyright:
+    Copyright (C) 2013-2022 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+
+    This software product is a proprietary product of Nvidia Corporation and its affiliates
+    (the "Company") and all right, title, and interest in and to the software
+    product, including all associated intellectual property rights, are and
+    shall remain exclusively with the Company.
+
+    This software product is governed by the End User License Agreement
+    provided with the software product.
+
+@author: Nasr Ajaj
+@date:   Jul 14, 2022
+"""
+
+import os
+import platform
+from http import HTTPStatus
+
+try:
+    from utils.utils import Utils
+    from utils.ufm_rest_client import UfmRestClient, HTTPMethods
+    from utils.args_parser import ArgsParser
+    from utils.config_parser import ConfigParser
+    from utils.logger import Logger, LOG_LEVELS
+    from utils.exception_handler import ExceptionHandler
+except ModuleNotFoundError as e:
+    if platform.system() == "Windows":
+        print("Error occurred while importing python modules, "
+              "Please make sure that you exported your repository to PYTHONPATH by running: "
+              f'set PYTHONPATH=%PYTHONPATH%;{os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))}')
+    else:
+        print("Error occurred while importing python modules, "
+              "Please make sure that you exported your repository to PYTHONPATH by running: "
+              f'export PYTHONPATH="${{PYTHONPATH}}:{os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))}"')
+
+
+class UfmPortsConstants:
+
+    PORTS_API_URL = 'resources/ports'
+    API_PARAM_SHOW_DISABLED = "show_disabled"
+    API_PARAM_SYSTEM = "system"
+    API_PARAM_ACTIVE = "active"
+
+    args_list = [
+        {
+            "name": f'--{API_PARAM_SYSTEM}',
+            "help": "Option to get all ports data related to specific system node"
+        },
+        {
+            "name": f'--{API_PARAM_ACTIVE}',
+            "help": "Option to get active ports only"
+        },
+        {
+            "name": f'--{API_PARAM_SHOW_DISABLED}',
+            "help": "Option to get disabled ports"
+        }
+    ]
+
+
+class UfmPortsConfigParser(ConfigParser):
+    def __init__(self, args):
+        super().__init__(args)
+
+
+class UfmPortsManagement:
+
+    @staticmethod
+    def get_ports(active, show_disabled, system=None):
+        url = f'{UfmPortsConstants.PORTS_API_URL}?{UfmPortsConstants.API_PARAM_ACTIVE}={active}&{UfmPortsConstants.API_PARAM_SHOW_DISABLED}={show_disabled}'
+        if system:
+            url = f'{url}&{UfmPortsConstants.API_PARAM_SYSTEM}={system}'
+        response = ufm_rest_client.send_request(url)
+        if response and response.status_code == HTTPStatus.OK:
+            Logger.log_message(response.json())
+        else:
+            Logger.log_message(response, LOG_LEVELS.ERROR)
```

Comment:
the response is object not string, and by the way no need to log the error here as we already have it in:
https://github.com/Mellanox/ufm_sdk_3.0/blob/main/utils/ufm_rest_client.py#L104